scripts/portstat

```diff
@@ -19,6 +21,9 @@ from natsort import natsorted
 from tabulate import tabulate
 from utilities_common.netstat import ns_diff, ns_brate, ns_prate, ns_util, table_as_json
 from utilities_common.intf_filter import parse_interface_in_filter
+from utilities_common.intf_filter import parse_interface_in_filter
+from utilities_common.multi_asic import DISPLAY_EXTERNAL, PORT_CHANNEL_OBJ, PORT_OBJ, multi_asic_args, run_on_all_asics
+from utilities_common.multi_asic import MultiAsic
```

Comment:
Suggest balnacing these imports out (3 per each line), or simply importing the entire module (`from utilities_common import multi_asic`), as long as there are many things in the module you won't use.

Reply:
how about the following format. Helps lines not being longer than 79 chars and groups imports together.
```
from utilities_common.multi_asic import (
    DISPLAY_EXTERNAL, 
    PORT_CHANNEL_OBJ, 
    PORT_OBJ, 
    multi_asic_args, 
    run_on_all_asics,
    MultiAsic
)
```

Reply:
This is better, but I feel like if you're importing the majority of content from a module, you should just import the entire module, as there's also less chance for namespace collisions.

Reply:
This file was added to this PR by mistake. Removed the changes to this PR